roles/dns/README.md

```diff
@@ -1,60 +1,146 @@
-## The rhc-ose ansible dns role
+Set of Roles
+============
 
+The ansible roles found in this directory has to do with managing DNS infrastructure.
 
-This role allows the caller to populate DNS records (currently only supports A records) based on input data (see example below). The caller may choose to use hostvars to access information from other hosts (e.g.: hostname and ip address) to create records.
+Requirements
+------------
 
-Both additions and removals are supported by this role, by the use of the `dns_records_rm` and `dns_records_add` variables (see example below).
+1. For named/bind install/configuration:  
+  1. Root level access to a running Linux flavor (Fedora, CentOS, Red Hat Enterprise Linux)
+1. For DNS zones and records:
+  1. Access to nsupdate enabled infrastructure (i.e.: above mentioned bind).
+  1. Access to AWS Route53 with DNS admin access enabled.
 
-> NOTE: removal of records is done before additions
 
+Role Variables
+--------------
 
-Example Playbook
+See `Example Inventory` below for more specific details. The following variable needs to be defined:
+
+- `dns_data`: A dictionary with DNS data (views, zones, records)
+
+
+Dependencies
```

Comment:
Are there any dependencies? I would assume having existing machines to run this against. If there aren't ( and we don't want to include the existing machines as a prereq) -- we should clean this up as part of a docs PR.

Reply:
Let's handle it as a follow-up docs PR